docs/features/multiple-networks/multi-homing.md

```diff
@@ -56,7 +108,8 @@ spec:
     }
 ```
 
-#### Network Configuration reference
+Network Configuration reference:
+
```

Comment:
I think this was better before ...

Reply:
switched back to `#### Network Configuration reference`